src/lib/viewers/doc/_docBase.scss

```diff
@@ -304,6 +304,17 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
         }
     }
 
+    .pdfViewer {
+        &--viewOnly {
```

Comment:
We typically want to spell out class names so they're easy to search for using our IDEs:

```
&.pdfViewer--viewOnly
```